app/screens/settings/display_theme/theme_thumbnail.tsx

```diff
@@ -0,0 +1,296 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import Svg, {Rect, G, Circle} from 'react-native-svg';
+
+type ThemeThumbnailProps = {
+    awayIndicator: string;
+    borderColorBase: string;
+    borderColorMix: string;
+    buttonBg: string;
+    centerChannelBg: string;
+    centerChannelColor: string;
+    dndIndicator: string;
+    newMessageSeparator: string;
+    onlineIndicator: string;
+    sidebarBg: string;
+    sidebarText: string;
+    sidebarUnreadText: string;
+    width: number;
+}
+
+function ThemeThumbnail({
+    awayIndicator = '#FFBC1F',
+    borderColorBase = '#E0E1E3',
+    borderColorMix = '#E0E1E3',
+    buttonBg = '#15B7B7',
+    centerChannelBg = 'white',
+    centerChannelColor = '#E0E1E3',
+    dndIndicator = '#D24B4E',
+    newMessageSeparator = '#1C58D9',
+    onlineIndicator = '#3DB887',
+    sidebarBg = '#174AB5',
+    sidebarText = '#86A1D9',
+    sidebarUnreadText = 'white',
```

Comment:
what are this default values and why do we need them? was this just copied from v1? I would want to know exactly why we have them here.

Reply:
I didn't refactor it enough after porting... mindless porting.  Thanks for catching this :)